test/runTests.cpp

```diff
@@ -37,34 +44,15 @@ int main(int argc, char* argv[]) {
     if (argc > 1) {
         numRandomTests = static_cast<int>(strtol(argv[1], NULL, 10));
     }
-
-    srand(42);
     bool allTestsPassed = true;
 
-    printf("Testing HW with alignment...\n");
-    allTestsPassed &= runRandomTests(numRandomTests, EDLIB_MODE_HW, true);
-    printf("\n");
-
-    printf("Testing HW...\n");
-    allTestsPassed &= runRandomTests(numRandomTests, EDLIB_MODE_HW, false);
-    printf("\n");
+    allTestsPassed &=
+            runRandomTestsForAllModes<char, unsigned char, 50>(numRandomTests, "char");
```

Comment:
I would maybe vote for not passing "char" to the function, but instead here having line like this (before `runRandomTestsForAllModes(...)`:
`printf("Running random tests for sequence of 'char', alphabet idx of 'unsigned char' and alphabet size of '50'");`